sdk-docs/testing-iris-apps/mock-out-graphql-calls.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Mocking out graphql calls
+category: 61fcd8e1a448f5004215317c
+parentDocSlug: testing-iris-apps
+---
+
+> 🚧 Beta
+> 
+> This is a beta version and subject to change without notice. Pricing, terms, conditions and availability may change in the final version.
+
+If you need to mock out graphql calls you can do it like this, lets say you are using a graphql call like this:
```

Comment:
Confusing sentence structure (talesprog). And would have seperated out the mocking into steps. Something like:

```
You might want to mock graphql calls for your tests. Consider the following GraphQL query as an example:
[query]

Then you can mock it like this:
[mock]

Notice, that we imported the following libraries:
[import statements]

- @testing-library/dom gives you access to...
- @trackuniot/react-core-context-tests helps you with...

[describe the test shortly]
```